scripts/artifacts-building/user_rcbops_artifacts_building.yml

```diff
@@ -31,3 +31,30 @@ openstack_repo_url: "{{ rpco_mirror_base_url }}"
 # default variant.
 lxc_image_cache_server: images.linuxcontainers.org
 
+# When we build python/container artifacts we pull the default
+# LXC cache variant from LXC upstream which may have packages
+# installed which are more recent than those we have available
+# in our artifacted apt repository. Here we figure out which
+# packages are installed, but not available in a configured
+# source, then we downgrade them to the latest available in
+# the configured sources.
+lxc_cache_prep_pre_commands: |
+    if [ -a /etc/resolv.conf ]; then
+      mv /etc/resolv.conf /etc/resolv.conf.org
+    fi
+    {% for resolver in lxc_cache_prep_dns %}
+    echo "nameserver {{ resolver }}" >> /etc/resolv.conf
+    {% endfor %}
+    apt-key add /root/repo.keys
+    export DEBIAN_FRONTEND=noninteractive
+    apt-get update
+    if apt list --installed 2>/dev/null | grep '\/now'; then
```

Comment:
I'd put a reference of what /now means somewhere.
I see on my machine /repo,now etc... The format would be good to explain to make sure it's doing what you're explaining it does.